### ISSUE TITLE
save_image is not working with CVaaS instance

### ISSUE THREAD

**Issue by titom73** [aristanetworks/cvprac#176]
## Issue summary

When using `cvprac.api.save_image` with a CVaaS instance, python fails and raises an error messaage related to APP_SESSION_ID

If same code is run against on premise instance, issue is not present.

## How to reproduce

Here is the code I use to reproduce Issue

```python
logging.info('File path to upload: {}'.format(image_path))
if os.path.exists(image_path) is False:
    logging.error('File not found ({}) !'.format(image_path))
    return False
image_path = os.path.abspath(image_path)
logging.info('File path to upload: {}'.format(image_path))
upload_result = None

# Check if existing image
# If image already exists, cancel it.
images = self._cvp_client.api.get_images()
logging.debug('List of images: {}'.format(images))
if images['total'] > 0:
    for image in images['data']:
        if image['name'] == os.path.basename(image_path):
            self._cvp_client.api.cancel_image(image['name'])
    images = self._cvp_client.api.get_images()
    logging.debug('List of images: {}'.format(images))

# try:
upload_result = self._cvp_client.api.add_image(filepath=image_path)
```

## Python traceback

```
unit/test_cvTools.py:52: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../eos_downloader/cvp.py:76: in upload_image
    upload_result = self._cvp_client.api.add_image(filepath=image_path)
../../../.pyenv/versions/3.9.4/envs/eos-download/lib/python3.9/site-packages/cvprac/cvp_api.py:2011: in add_image
    response = self.clnt.post('/image/addImage.do',
../../../.pyenv/versions/3.9.4/envs/eos-download/lib/python3.9/site-packages/cvprac/cvp_client.py:875: in post
    return self._make_request('POST', url, timeout, data=data, files=files)
../../../.pyenv/versions/3.9.4/envs/eos-download/lib/python3.9/site-packages/cvprac/cvp_client.py:612: in _make_request
    response = self._send_request(req_type, full_url, timeout,
                                  
                    else:
                        fhs = dict()
                        fhs['Accept'] = self.headers['Accept']
>                       fhs['APP_SESSION_ID'] = self.headers['APP_SESSION_ID']
E                       KeyError: 'APP_SESSION_ID'

../../../.pyenv/versions/3.9.4/envs/eos-download/lib/python3.9/site-packages/cvprac/cvp_client.py:731: KeyError
```

**Comment by mharista** [comment]
I'll look into fixing this. Hoping a simple check for APP_SESSION_ID in headers similar to how the Authorization key is handled will resolve this.

**Comment by titom73** [comment]
I thought the same, but just not sure why we have this difference and if it is expected or not. For information, I use `staging` instance of CVaaS.

**Comment by mharista** [comment]
PR #177 created to fix this. Issue reproduced and solution tested with prod CVaaS.

**Comment by mharista** [comment]
Fixed #177
